### PR TITLE
Pda 45

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: widgets/src/api/pda/connections.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
 
-import MOCK_CONNECTIONS from 'mocks/pda/connections';
+import MOCK_CONNECTIONS, {
+  TEST_CONNECTION,
+  SAVE_CONNECTION,
+  DELETE_CONNECTION,
+} from 'mocks/pda/connections';
 import makeRequest from 'api/makeRequest';
 
 // eslint-disable-next-line import/prefer-default-export
```
Comment (rimselis) [inline widgets/src/api/pda/connections.js]: not needed anymore
```suggestion

```

File: widgets/src/components/Connections/ConnectionsContainer.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+              onSave={this.handleSave}
+            />
+          </WidgetBox>
+
+          <Grid className={classes.gridContainer} container spacing={2}>
+            {listLoader &&
+              Array(4)
+                .fill('')
+                // eslint-disable-next-line react/no-array-index-key
+                .map((_, i) => <ConnectionItemSkeleton key={i} />)}
```
Comment (rimselis) [inline widgets/src/components/Connections/ConnectionsContainer.js]:
```suggestion
            { listLoader && (
              <>
                <ConnectionItemSkeleton />
                <ConnectionItemSkeleton />
                <ConnectionItemSkeleton />
                <ConnectionItemSkeleton />
              </>
            )}
```

File: widgets/src/components/Connections/ConnectionsContainer.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
+/* eslint-disable no-console */
+/* eslint-disable react/jsx-wrap-multilines */
```
Comment (rimselis) [inline widgets/src/components/Connections/ConnectionsContainer.js]: I suggest moving this to affected line
```suggestion
```

File: widgets/src/components/Connections/ConnectionsContainer.js [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    const { classes } = this.props;
+    const { connectionsList, showForm, formData, listLoader, loader, notification } = this.state;
+
+    return (
+      <ThemeProvider theme={theme}>
+        <Container>
+          <WidgetBox
+            title="Connections"
+            topRightComp={
+              <IconButton color="primary" onClick={this.handleAddButton}>
```
Comment (rimselis) [inline widgets/src/components/Connections/ConnectionsContainer.js]:
```suggestion
              // eslint-disable-next-line react/jsx-wrap-multilines
              <IconButton color="primary" onClick={this.handleAddButton}>
```

**Comment by entando-service** [comment]
All is well

**Review by rimselis — DISMISSED** [review]

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Review by rimselis — DISMISSED** [review]

**Comment by entando-service** [comment]
All is well

**Review by rimselis — APPROVED** [review]

**Pull request merged**